One component is visible on home page.

developer-ishan opened the pull request:
# Pull Request Template

## Description
One component is visible on home page.

Fixes # (issue)
Navbar logo made responsive

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My contribution follows the style guidelines of this project
- [x] I have performed a self-review of my own contribution
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

nehakalbande commented:
Hi @developer-ishan 
- Please remove the bg color of Vision Test in Home Page
- In navbar logo position is changed, whole Logo should be in the navbar
- Due to the changes, navbar is not working in mobile responsive

developer-ishan commented:
-  The mistake was in the states of NavItems.js.
I never touched that file but have corrected the bug anyway
-  Removed the color
- Corrected the height

I am making a fresh PR please accept that.
